src/pfs/pfs.proto

```diff
@@ -284,6 +294,31 @@ message DeleteBranchRequest {
   bool force = 2;
 }
 
+message CreateProjectRequest {
+  Project project = 1;
+  string description = 2;
+  bool update = 3;
```

Comment:
What are the semantics of `update` here? Is updating a project name allowed? Are all repos in the project updated? Ditto all pipelines and pipeline inputs?